marshmallow_dataclass/__init__.py

```diff
@@ -60,8 +60,6 @@ class User:
 
 from typing import TypedDict
 
-from blib.attributes import SerializableAttribute, SerializableAttributeTag
```

Comment:
there would be a circular dependency if blib/specs needed marshmallow_dataclass, but marshmalow_dataclass imported these types